sensor/extrinsic_reflector_based_calibrator/src/extrinsic_reflector_based_calibrator.cpp

```diff
@@ -1389,7 +1539,17 @@ void ExtrinsicReflectorBasedCalibrator::visualizationMarkers(
   }
 
   matches_markers_pub_->publish(matches_marker_array);
+}
 
+void ExtrinsicReflectorBasedCalibrator::visualizeTrackMarkers()
```

Comment:
nit: these lambas are often used when you only used them locally once.
If you end up reusing them (code duplication) it is preferable to have a header file for them

Reply:
lambada function `auto eigen_to_point_msg ` now change to function `geometry_msgs::msg::Point ExtrinsicReflectorBasedCalibrator::eigenToPointMsg(const Eigen::Vector3d &p_eigen) ` in commit https://github.com/tier4/CalibrationTools/pull/145/commits/82d940da6f5dfdebeea22fd0649d4526e63a8c7f